### ISSUE TITLE
Support className property for shim experience without the shim

### ISSUE THREAD

**Issue by sastan** [tw-in-js/twind-jsx-preprocessor#7]
```jsx
<span className="font-bold">Blue</span>
```

Output

```jsx
import { tw as _tw, directive as _directive } from 'twind'

<span className={_tw`font-bold`}>Blue</span>
```

References:

- https://github.com/tw-in-js/use-twind-with/tree/main/packages/preact#readme
- https://github.com/tw-in-js/use-twind-with/blob/main/packages/preact/index.ts#L71

**Comment by itsMapleLeaf** [comment]
I'm not really sold on this 🤔 `className="..."` is just `tw="..."` but longer, less capable, and with the potential of unwarranted side-effects. I think `className` should stay as-is for passing literal, unfiltered class names if ever needed

**Comment by sastan** [comment]
You are right. It is meant as an alternative shim (no runtime) that needs to be enabled explicitly.
